modules/core/src/main/java/org/apache/ignite/internal/commandline/diagnostic/DiagnosticCommand.java

```diff
@@ -101,20 +103,28 @@
 
     /** {@inheritDoc} */
     @Override public String name() {
-        return "Diagnostic command";
+        return "diagnostic";
     }
 
     /** {@inheritDoc} */
     @Override public void printUsage() {
-        logger.info("View diagnostic information in a cluster. For more details type:");
-        logger.info(join(" ", UTILITY_NAME, DIAGNOSTIC, HELP));
-        logger.info("\n");
+        Command.usage("View diagnostic information in a cluster:", DIAGNOSTIC);
     }
 
     /**
      *
      */
     private void printDiagnosticHelp() {
-        logger.info(join(" ", UTILITY_NAME, DIAGNOSTIC, PAGE_LOCKS + " - dump page locks info."));
+        CommandLogger.logWithIndent(join(" ", UTILITY_NAME, DIAGNOSTIC, PAGE_LOCKS + " - dump page locks info."));
+
+        CommandLogger.nl();
+
+        CommandLogger.logWithIndent("Subcommands:");
+
```

Comment:
Why do you use `CacheCommandList`? Diagnostics doen't relates with `--cache` commands